### PR TITLE
Host yards vcr

### LINKED ISSUE #69: Host Yards - Implement VCR and OS
Refactor Host Yards method to implement VCR and call to real endpoint to confirm OpenStruct object is correct

### PR DESCRIPTION
**Pull request by amsmyth1**
**What Changed** 
- Added VCR path to host_yards feature test. 
- Added Sad path stub testing to host_yards feature test
- Added Openstruct to host_yards method
- Added Dashboard facade to create openstruct for the host/dashboard controller

close #69 

**What's Next** 
- Yard Show Page
- Switching local3001 endpoint to heroku app endpoint. 

** Known Issues **
- Currently all tests pass when local3001 YSengine is being used for endpoints, but heroku URL does not work. Multiple parse token JSON errors. Team needs to review this.

### CODE REVIEW

File: app/views/host/dashboard/index.html.erb [hunk 1]
```diff
@@ -6,17 +6,18 @@
...
-      <%= yard[:attributes][:price] %>
-      <% yard[:attributes][:purposes][:data].each do |purpose_object| %>
-        <% purpose_object[:attributes][:name] %>
+    <section id="yard-<% yard.id%>">
+      <%= yard.name %>
+      <%= yard.description %>
+      <%= yard.address %>
+      <%= yard.price %>
+      <% yard.purposes.each do |purpose| %>
+        <%= purpose %>
```
Comment (jbranham1) [inline app/views/host/dashboard/index.html.erb:19]: Looks much cleaner this way!

File: app/facades/dashboard_facade.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class DashboardFacade
```
Comment (jbranham1) [inline app/facades/dashboard_facade.rb:1]: This facade looks great!

**Review by jbranham1 — APPROVED** [review]
Great job on this refactor and use of VCR!

**Review by dev-named-jordan — APPROVED** [review]
This looks really great! Dashboard index view is looking really clean!

**Pull request merged**